### ISSUE TITLE
AppIcon contents.json generation

### ISSUE THREAD

**Issue by mpsnp** [e-sites/Natrium#10]
Here is the problem: if i entirelly ignore and remove `AppIcon.appiconset` folder, then natrium fails to generate it. but if i commit only `contents.json` inside of it, natrium doesn't use ordering while generating this json, and it changes each time it is generated.

What's the best practice of using natrium with AppIcon?

**Comment by basvankuijck** [comment]
And what if you put `Project/Resources/Assets.xcassets/AppIcon.appiconset/*.png`in your `.gitignore` ?
But keep the `Contents.json`

**Comment by mpsnp** [comment]
@basvankuijck natrium reorders keys inside of `Contents.json` each time it is generated. If you use codable for that, it supports sorting alphabetically as i remember (but it works inside of objects). In order to keep "images" contents pretty, we need to sort this array somehow too.

**Comment by basvankuijck** [comment]
@mpsnp: I saw that too. `v6.0.2` should have fixed that

**Comment by mpsnp** [comment]
@basvankuijck are you sure, that you don't need to add extra option [here](https://github.com/e-sites/Natrium/blob/master/Sources/Parsers/AppIconParser.swift#L228) for sorting?

**Comment by basvankuijck** [comment]
> @basvankuijck are you sure, that you don't need to add extra option [here](https://github.com/e-sites/Natrium/blob/master/Sources/Parsers/AppIconParser.swift#L228) for sorting?

Yeah that should work. You can also remove the directory entirely and git ignore `Project/Resources/Assets.xcassets/AppIcon.appiconset/` completely.

Because the parser now checks if the directory exists and creates it if necessary:
https://github.com/e-sites/Natrium/blob/master/Sources/Parsers/AppIconParser.swift#L58
